tdrs-backend/tdpservice/parsers/models.py

```diff
@@ -90,6 +90,9 @@ class Status(models.TextChoices):
 
     case_aggregates = models.JSONField(null=True, blank=False)
 
+    total_number_of_records_in_file = models.IntegerField(null=True, blank=False, default=0)
+    total_number_of_records_created = models.IntegerField(null=True, blank=False, default=0)
```

Comment:
what's the difference between these two? is it possible a record is added to `total_number_of_records_in_file` but doesn't get counted in `total_number_of_records_created`?

Reply:
`total_number_of_records_in_file` is the total number of records that were counted in the file, that includes multi-record schemas. `total_number_of_records_created` is the number of records we serialized to PG. The two numbers won't always be the same. This is because records that have pre-check validation errors aren't created.